WordPress/Classes/ViewRelated/Plans/PlanComparisonViewController.swift

```diff
@@ -44,14 +44,20 @@ class PlanComparisonViewController: UIViewController {
         return button
     }()
     
-    class func controllerWithInitialPlan(plan: Plan, activePlan: Plan? = nil, planService: PlanService) -> PlanComparisonViewController {
+    class func controllerWithInitialPlan(plan: Plan, activePlan: Plan? = nil, siteID: Int, planService: PlanService<StoreKitStore>) -> PlanComparisonViewController {
         let storyboard = UIStoryboard(name: "Plans", bundle: NSBundle.mainBundle())
         let controller = storyboard.instantiateViewControllerWithIdentifier(NSStringFromClass(self)) as! PlanComparisonViewController
 
+        // @koke 2016-03-15
+        // This is very fragile. If we set siteID after currentPlan it will crash.
```

Comment:
As we discussed a while ago, it'd be nice if we can look at avoiding this via a view model or something.